pages/publishing/importing.mdx

```diff
@@ -160,9 +160,10 @@ You can learn more about common Tracking Plan spreadsheet formats [on our blog.]
 
 ##### Optional columns
 
-| Column Name  | Description                                    | Example value |
-| :----------- | :--------------------------------------------- | :------------ |
-| Pinned Value | The pinned value of the property on this event | iOS           |
+| Column Name  | Description                                                       | Example value |
+| :----------- | :---------------------------------------------------------------- | :------------ |
+| Event Tags   | The tags attached to your event (separate multiple tags with ';') | Owner:Growth  |
```

Comment:
Wait are they separated with `;` or `:`?

Reply:
the importer supports `;` and `\n`

Reply:
Okay. I ask because the example says `Owner:Growth` and it makes it seem like these are two tags but they'd be imported as a single tag, right? Should we change the example to include a semi-comma?

Reply:
We could do two values here like "Owner:Growth; Sprint:23" or something of the sort?

Reply:
Also @logason I've duplicated the Ultimate Tracking Plan template and made the necessary changes to the new copy, which you can find at https://docs.google.com/spreadsheets/d/1MpoYMby6sTR9X9qavmUtpFmGIxr79I4f_kdv6OobvYQ (replacement for link in line 144 for "View example sheet"). I can't change its visibility to Public though - would you be able to replace the link and change the visibility of the sheet to Public and not Avo?